### ISSUE TITLE
Section Content Descriptions 

### ISSUE THREAD

**Issue by andreamclemeno** [parsamorsal/Inverted-Pendulum#12]
Hi Parsa!

I know I said to delete the comments earlier but I think it would be good to have some description for each of the following sections: Terminology and Definitions, Functional Requirements, Nonfunctional Requirements, Likely and Unlikely Changes. 

I think Dr. Smith said we could write something similar or use the descriptions given by the SRS template so you can use those!!

Andrea

@parsamorsal

**Comment by smiths** [comment]
Yes, you can definitely borrow any text you would like from the SRS template document.  :smile:

**Comment by parsamorsal** [comment]
Solved. dd96397
